src/services/expenses.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const { Expense } = require('../models/expenses');
+
+const getAll = async(filterOptions) => {
+  const expenses = await Expense.findAll();
```

Comment:
Apply the filterOptions directly in the Expense.findAll() to reduce the number of fetched expenses.